### PR TITLE
MGMT-2130 Delete permissions for leases

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: deploy/roles/default_role.yaml [hunk 1]
```diff
@@ -45,12 +45,6 @@ rules:
     resources:
       - configmaps
     resourceNames: ["assisted-service-leader-election-helper"]
-  - verbs:
```
Comment (filanov) [inline deploy/roles/default_role.yaml:48]: What does it affect?

Comment (tsorya) [reply deploy/roles/default_role.yaml:48]: nothing

Comment (tsorya) [reply deploy/roles/default_role.yaml:48]: that's why i remove it. Looks like it was bug to add it

Comment (filanov) [reply deploy/roles/default_role.yaml:48]: Did you check it on PSI? (that it's not breaking anything)

Comment (tsorya) [reply deploy/roles/default_role.yaml:48]: sure, i did

Comment (filanov) [reply deploy/roles/default_role.yaml:48]: Great. 
What about app-sre? do we need to change something there? or ask someone to remove this permission?

**Comment by filanov** [comment]
FYI @yboaron

File: deploy/roles/default_role.yaml (hunk 1 above)
Comment (ybettan) [reply deploy/roles/default_role.yaml:48]: I am on the app-sre side @filanov

Comment (ybettan) [reply deploy/roles/default_role.yaml:48]: https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/9126
Waiting for this PR to be merged before removing the `/hold` from the MR.

**Review by filanov — APPROVED** [review]

**Pull request merged**